### PR TITLE
Add example for port mapping

### LINKED ISSUE #19: Describe the method for getting the value of the cluster ports in README
This will be useful for testing REST API

### PR DESCRIPTION
**Pull request by nickkkccc**
I haven't forgotten about:
- [x] Tests
- [ ] Changelog
- [x] Documentation
- [x] Commit messages comply with the [guideline](https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message)
- [x] Cleanup the code for review. See [checklist](https://github.com/tarantool/cartridge-java/blob/master/docs/review-checklist.md)

Related issues:
Closes #19

### CODE REVIEW

**Review by dkasimovskiy — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -193,6 +193,76 @@ public class SomeOtherTest {
...
 
+##### Mapping ports
+
+Often there is a need to connect to a container via a certain port. To achieve this goal, you need to know the mapped 
+port of the port that was specified from the Java code. To get the mapped port use the `getMappedPort(...)` method
+of testcontainers API. 
+
+As an example, consider the following Java code:
+
+```java
```
Comment (ArtDu) [inline README.md:204]: It's better to have it in test code and put the link here

File: README.md [hunk 2]
```diff
@@ -193,6 +193,76 @@ public class SomeOtherTest {
...
+
+public class AppTest {
+
+    private final Logger logger = LoggerFactory.getLogger(AppTest.class);
+
+    private static final GenericContainer<?> container =
+            new TarantoolCartridgeContainer("cartridge/instances.yml", "cartridge/topology.lua")
+                    .withDirectoryBinding("cartridge")
+                    .withRouterHost("localhost")
+                    .withRouterPort(3301)
```
Comment (ArtDu) [inline README.md:225]: It's better to add an example about other ports. Not only about the main 3301. What if I want to connect to another router

**Review by ArtDu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nickkkccc) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (nickkkccc) [reply README.md]: Fixed

File: pom.xml [hunk 3]
```diff
@@ -92,7 +92,12 @@
         </dependency>
 
         <!-- Test dependencies -->
-        <dependency>
+      <dependency>
+        <groupId>org.apache.httpcomponents</groupId>
```
[the remark this answers is not in the document]
Comment (nickkkccc) [reply pom.xml:96]: Fixed

File: src/test/java/org/testcontainers/containers/TarantoolCartridgePortMappingTest.java [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+        final StringBuilder curlCommandToConnectToRouters = new StringBuilder("http://localhost:")
+                .append(container.getMappedPort(httpPortToFirstRouter));
+
+        // send get request to first router via http
+        HttpResponse response = sendCurlToRouterHttpAPI(curlCommandToConnectToRouters.toString());
+        assertEquals(200, response.getStatusLine().getStatusCode());
+
+        curlCommandToConnectToRouters.delete(0, curlCommandToConnectToRouters.length())
+                                     .append("http://localhost:")
```
[the remark this answers is not in the document]
Comment (nickkkccc) [reply src/test/java/org/testcontainers/containers/TarantoolCartridgePortMappingTest.java]: Replaced

File: pom.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (dkasimovskiy) [reply pom.xml:96]: Keep formatting the same as for dependencyManagement section.
Check your settings for tab indention.

File: src/test/java/org/testcontainers/containers/TarantoolCartridgePortMappingTest.java [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+        final int httpPortToFirstRouter = 8081;
+        final int httpPortToSecondRouter = 8082;
+        final int portToFirstRouter = 3301;
+        final int portToSecondRouter = 3302;
+        final String host = "localhost";
+
+        container.addExposedPorts(httpPortToFirstRouter, httpPortToSecondRouter, portToFirstRouter, portToSecondRouter);
+        container.start();
+
+        URI uri = new URI("http", null, host, container.getMappedPort(httpPortToFirstRouter), null,
```
Comment (dkasimovskiy) [inline src/test/java/org/testcontainers/containers/TarantoolCartridgePortMappingTest.java:51]: Use more specific name for variable.
It's bad practice to use generic names for a variables.

File: src/test/java/org/testcontainers/containers/TarantoolCartridgePortMappingTest.java [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+                null, null);
+
+        // send get request to first router via http
+        HttpResponse response = sendCurlToRouterHttpAPI(uri);
+        assertEquals(200, response.getStatusLine().getStatusCode());
+
+        uri = new URI("http", null, host, container.getMappedPort(httpPortToSecondRouter), null,
```
Comment (dkasimovskiy) [inline src/test/java/org/testcontainers/containers/TarantoolCartridgePortMappingTest.java:58]: Bad idea to use the one variable for different instances.

**Review by dkasimovskiy — CHANGES_REQUESTED** [review]

File: src/test/java/org/testcontainers/containers/TarantoolCartridgePortMappingTest.java (hunk 5 above)
Comment (nickkkccc) [reply src/test/java/org/testcontainers/containers/TarantoolCartridgePortMappingTest.java]: Fixed

File: README.md [hunk 7]
```diff
@@ -232,6 +232,13 @@ cartridge build at the image build stage.
 
 An example of how to set the `TARANTOOL_CLUSTER_COOKIE` parameter: https://github.com/tarantool/testcontainers-java-tarantool/blob/355d1e985bd10beca83bc7ca77f919a288709419/src/test/java/org/testcontainers/containers/TarantoolCartridgeBootstrapFromLuaWithFixedPortsTest.java#L57-L82
 
+##### Mapping ports
+
+Often there is a need to connect to a container through a specific port. To achieve this goal it is necessary
+know the mapped port specified in the Java code. To get the mapped port, use the getMappedPort(...)` method of
+testcontainers API. See examples: https://github.com/tarantool/testcontainers-java-tarantool/blob/ae580cb59d4621d7c11e5f80c394ec141d4c92b3/src/test/java/org/testcontainers/containers/TarantoolCartridgePortMappingTest.java
```
Comment (ArtDu) [inline README.md:239]: The link has to have line anchors. Check render after pushing

File: README.md [hunk 8]
```diff
@@ -232,6 +232,13 @@ cartridge build at the image build stage.
 
 An example of how to set the `TARANTOOL_CLUSTER_COOKIE` parameter: https://github.com/tarantool/testcontainers-java-tarantool/blob/355d1e985bd10beca83bc7ca77f919a288709419/src/test/java/org/testcontainers/containers/TarantoolCartridgeBootstrapFromLuaWithFixedPortsTest.java#L57-L82
 
+##### Mapping ports
+
+Often there is a need to connect to a container through a specific port. To achieve this goal it is necessary
+know the mapped port specified in the Java code. To get the mapped port, use the getMappedPort(...)` method of
```
Comment (ArtDu) [inline README.md:238]:
```suggestion
Often there is a need to connect to a container through a specific port. To achieve this goal it is necessary
to know the mapped port specified in the Java code. To get the mapped port, use the getMappedPort(...)` method of
```

**Review by ArtDu — CHANGES_REQUESTED** [review]

File: README.md (hunk 7 above)
Comment (nickkkccc) [reply README.md]: Fixed

File: README.md (hunk 8 above)
Comment (nickkkccc) [reply README.md]: Fixed

**Review by dkasimovskiy — APPROVED** [review]

**Review by ArtDu — APPROVED** [review]

**Pull request merged**